File: JavaSide/src/java/entity/Address.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity;

import okhttp3.*;
import org.json.*;

/**
 *
 * @author xy
 */
public class Address {

    private double longitude;
    private double latitude;
    private String locationName;
    private String area;

    /**
     *
     * @param locationName
     * @param longitude
     * @param latitude
     * @param area
     */
    public Address(String locationName, double longitude, double latitude, String area) {
        setLocationName(locationName);
        setLongitude(longitude);
        setLatitude(latitude);
        setArea(area);
    }

    /**
     *
     * @return
     */
    public double getLongitude() {
        return longitude;
    }

    /**
     *
     * @return
     */
    public double getLatitude() {
        return latitude;
    }

    /**
     *
     * @return
     */
    public String getLocationName() {
        return locationName;
    }

    /**
     *
     * @return
     */
    public String getArea() {
        return area;
    }

    /**
     *
     * @param longitude
     */
    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    /**
     *
     * @param latitude
     */
    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    /**
     *
     * @param locationName
     */
    public void setLocationName(String locationName) {
        this.locationName = locationName;
    }

    /**
     *
     * @param area
     */
    public void setArea(String area) {
        this.area = area;
    }

}

File: ASPWebsite/ASPWebsite/App_code/Entity/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ASPWebsite.App_Code.Entity
{
    /// <summary>
    /// Address.
    /// </summary>
    public class Address
    {
        public double longitude;
        public double latitude;
        public string locationName;
        public string area { get; set; }

        /// <summary>
        /// Initializes a new instance of the <see cref="T:ASPWebsite.App_Code.Entity.Address"/> class.
        /// </summary>
        /// <param name="locationName">Location name.</param>
        /// <param name="longitude">Longitude.</param>
        /// <param name="latitude">Latitude.</param>
        /// <param name="area">Area.</param>
        public Address(string locationName, double longitude, double latitude, string area)
        {
            setLocationName(locationName);
            setLongitude(longitude);
            setLatitude(latitude);
            setArea(area);
        }
        /// <summary>
        /// Gets the longitude.
        /// </summary>
        /// <returns>The longitude.</returns>
        public double getLongitude()
        {
            return longitude;
        }
        /// <summary>
        /// Gets the latitude.
        /// </summary>
        /// <returns>The latitude.</returns>
        public double getLatitude()
        {
            return latitude;
        }
        /// <summary>
        /// Gets the name of the location.
        /// </summary>
        /// <returns>The location name.</returns>
        public string getLocationName()
        {
            return locationName;
        }
        /// <summary>
        /// Gets the area.
        /// </summary>
        /// <returns>The area.</returns>
        public string getArea()
        {
            return area;
        }
        /// <summary>
        /// Sets the longitude.
        /// </summary>
        /// <param name="longitude">Longitude.</param>
        private void setLongitude(double longitude)
        {
            this.longitude = longitude;
        }
        /// <summary>
        /// Sets the latitude.
        /// </summary>
        /// <param name="latitude">Latitude.</param>
        private void setLatitude(double latitude)
        {
            this.latitude = latitude;
        }
        /// <summary>
        /// Sets the name of the location.
        /// </summary>
        /// <param name="locationName">Location name.</param>
        private void setLocationName(string locationName)
        {
            this.locationName = locationName;
        }
        /// <summary>
        /// Sets the area.
        /// </summary>
        /// <param name="area">Area.</param>
        private void setArea(string area)
        {
            this.area = area;
        }
    }
}